extension-api/app/api/routes.py

```
@@ -123,55 +144,79 @@ async def test_db():
+            "received_header": (
+                authorization[:20] + "..."
+                if authorization and len(authorization) > 20
+                else authorization
+            ),
             "header_present": authorization is not None,
-            "starts_with_bearer": authorization.startswith("Bearer ") if authorization else False
+            "starts_with_bearer": (
+                authorization.startswith("Bearer ") if authorization else False
+            ),
         }
     except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)